emmo/emmocheck.py

```diff
@@ -281,6 +283,10 @@ def main():
     parser.add_argument(
         '--url-from-catalog', '-u', action='store_true',
         help=('Get url from catalog file'))
+    parser.add_argument(
+        '--ignore-namespace', '-in', action='append', default=[],
```

Comment:
Unusual with 2-letter short option. Normally -in is a shortcut for "-i -n" if "-i" doesn't take an argument or "-i n" if "-i" takes an argument.

Reply:
thank you, this is fixed